src/main/resources/suite.xml

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+<suite name="test examples"
+       allow-return-values="true" verbose="2"
+       parallel="tests">
+    <test name="all" parallel="classes" thread-count="5" time-out="240000">
```

Comment:
Test will fail if toimeout timne will end, so try to test with 1 second, for example